tests/integration/device_test.sh

```diff
@@ -4,6 +4,11 @@ include './tests/unit/utils.sh'
 include './tests/integration/utils.sh'
 include './src/device_info.sh'
 
+function oneTimeSetUp()
+{
+  prepare_testing_environment 'device'
```

Comment:
With this function, we display the setup for every test, for example:

```
Preparing environment for kw config integration tests...
[1/6] Using cached container image for archlinux.
[2/6] Creating archlinux container.
[3/6] Using cached container image for debian.
[4/6] Creating debian container.
[5/6] Using cached container image for fedora.
[6/6] Creating fedora container.
```

Can we add a silent option for the `setup_container_environment` function?

Reply:
I've included a `--verbose` option for situations where we need to see this information.